### ISSUE TITLE
Instead of just review counts, break down PRs into reviewers & authors.

### ISSUE THREAD

**Issue by mattpolzin** [mattpolzin/harmony#17]
Count both authored and review requested PRs. Not entirely sure how to incorporate authored counts into harmony's heuristic for review assignment, but harmon can definitely show some informative data under the graph command by showing authored counts as well as review request counts.

Equally as important, this is maybe a good time to switch from requesting open and closed PRs in separate http requests to instead making 1 request and passing whether the PR is open or closed back to Idris.

In other words, Idris will go from aggregating reviewers for open PRs and then closed PRs to aggregating reviewers, authors, and open/closed state across all PRs with a single HTTP request.
